### ISSUE TITLE
Running-config link may expose secret environment variables

### ISSUE THREAD

**Issue by hznuyyh** [cloudprober/cloudprober#534]
When I use the Runing-config route to view the runtime configuration, the env I set will be displayed, which can lead to some security risks.
For example, when I set up some access account/password/token for some http probe, it should not be visible.

```eg.
   http_probe {
    relative_url: "/api/xxxxx"
    method: GET
    protocol: HTTPS
    headers {
      name: "Authorization"
      value: "{{env "TOKEN"}}"
    }
  }
```

Can config-running be set to hide as an optional route, or the set env variable is not displayed?

**Comment by manugarg** [comment]
I think this is a genuine concern. I've thought about it a few times but never got around to filing an issue for it to track it. Thanks @hznuyyh!

We can provide an option to either remove the "/config-running" page completely (easy) or just hide the environment variables on that page. I'll think more about it.

**Comment by manugarg** [comment]
#536 with the possible solution.

**Comment by manugarg** [comment]
Fixed by #536.
